internal/services/sarif/sarif.go

```diff
@@ -190,3 +190,7 @@ func (s *Sarif) getSarifSeverityMap() map[severities.Severity]string {
 		severities.Info:     Note,
 	}
 }
+
+func (s *Sarif) getVersionWithoutPrefix() string {
+	return strings.ReplaceAll(version.Version, "v", "")
```

Comment:
What do you think about? `return version.Version[1:]`

Reply:
This will lead to a strange output for the local build binaries, since we use the `{{VERSION_NOT_FOUND}}` text.

Reply:
Make sense.